Plugins/Flow.Launcher.Plugin.Explorer/Main.cs

```diff
@@ -32,7 +33,12 @@ public async Task InitAsync(PluginInitContext context)
             viewModel = new SettingsViewModel(context);
             await viewModel.LoadStorage();
             Settings = viewModel.Settings;
-            contextMenu = new ContextMenu(Context, Settings);
+
+            // as at v1.7.0 this is to maintain backwards compatibility, need to be removed afterwards.
+            if (Settings.QuickFolderAccessLinks.Any())
+                Settings.QuickAccessLinks = Settings.QuickFolderAccessLinks;
```

Comment:
Please also remove QuickFolderAccessLinks so that it won't be saved via json.

Reply:
Good pick up 👍 
It needs to be loaded once, so I dont know how I can make it so it gets loaded but ignored when saving. Made it null so it's saved as empty list.